app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
```

Comment:
Please correct messages for Exceptions: For example: `raise NotVaccinatedError("Visitor must be vaccinated")`